### ISSUE TITLE
Accessibility: Message history can no longer be read with screen readers

### ISSUE THREAD

**Issue by winman3000** [miranda-ng/miranda-ng#3121]
# Accessibility: message history can no longer be read with screen readers

## Description.

When I want to read the message history with stdmsg (version 0.96.2.24648), I can no longer read it with JAWS or Narrator.

## Steps to reproduce

The following steps can be easily reproduced with the included screen reader Narrator.

1. Run Narrator (CTRL+Windows key+Enter).
1. Select a contact in Miranda NG and confirm with Enter. However, make sure you are working with STDMSG.
1. Write a message if necessary and send it with Enter.
1. Go to message history with TAB and press arrow up and down to read message history.

### Result

The message history is not readable. For screen readers, it is presented as blank.

### Expected

The message history should be readable with arrow up and down.

**Comment by georgehazan** [comment]
@winman3000 do you use stable version? which SRMM plugin?

**Comment by winman3000** [comment]
@[georgehazan](https://github.com/georgehazan): No, I am using the development version. When I use the stdmsg.dll version 0.96.2.24631, it works. With the latest version, it doesn't.

**Comment by georgehazan** [comment]
@winman3000 try to turn on Options - Message sessions - Tabs - Enable
I'll try to fix it
